web-admin/src/routes/+page.svelte

```diff
@@ -22,26 +22,24 @@
 </svelte:head>
 
 <AuthRedirect>
-  <section class="flex flex-col w-4/5 mx-auto h-2/5">
-    <h1 class="text-4xl leading-10 font-light mb-2 my-auto">
+  <section class="flex flex-col w-4/5 mx-auto h-5/6 my-36">
```

Comment:
It appears this use of `h-5/6` may be creating a scrollbar. It'd be a good idea to _not_ set the height like this; it's pretty unstable.

![Kapture 2023-05-03 at 20 35 41](https://user-images.githubusercontent.com/95735/236107425-6551ec19-a9e6-4fd9-b188-07a189c5a6c6.gif)

Reply:
additionally, it would be a good idea to add some gap between the two columns, since in slightly smaller screens the left column looks too close to the right column.

Reply:
Yeah good call – just went with a responsive margin approach